### PR TITLE
Add MR comments to Shortages Planner Report per Item

### PR DESCRIPTION
**Pull request by RossStewart**
Updated the View to include the MR comments. Minor changes to Planner report to include the comments on the end of the item.

### CODE REVIEW

File: src/Domain.LinnApps/Reports/ShortagesReportService.cs [hunk 1]
```diff
@@ -127,6 +127,7 @@ public IEnumerable<ResultsModel> GetShortagesPlannerReport(int planner)
                     model.AddColumn("TotalBTReqt", "To achieve Ideal Build");
                     model.AddColumn("Supplier", "Supplier");
                     model.AddColumn("SupplierName", "Supplier Name");
+                    model.AddColumn("PlannerStory", "Planner Story");
```
Comment (richardiphillips) [inline src/Domain.LinnApps/Reports/ShortagesReportService.cs:130]: Don't call it planner story on the actual report. That's just what the table column happens to be called but they won't know what it is

**Review by richardiphillips — APPROVED** [review]
do you need me to add planner_story to the database view?

File: src/Domain.LinnApps/Reports/ShortagesReportService.cs (hunk 1 above)
Comment (RossStewart) [reply src/Domain.LinnApps/Reports/ShortagesReportService.cs:130]: Noted! Will change it

**Comment by RossStewart** [comment]
> do you need me to add planner_story to the database view?

all sorted!

**Pull request merged**